src/mapOfPeople.js

```diff
@@ -39,8 +39,22 @@
  *
  * @return {Map}
  */
-function mapOfPeople(people) {
-  // write code here
+function mapOfPeople(people = []) {
+  const uniquePeople = people.reduce((uniquePeople, person) => {
```

Comment:
https://github.com/mate-academy/js_n-for-names/pull/21#discussion_r275247892
`uniquePeople `